### ISSUE TITLE
X11 IME tooltip incorrectly placed with OpenGL backend

### ISSUE THREAD

**Issue by tdaede** [sixtyfpsui/sixtyfps#511]
```
cargo run --bin gallery
```
![Screenshot from 2021-09-18 22-02-38](https://user-images.githubusercontent.com/1308314/133916055-9f220651-04db-4b23-9b62-d7f2ef5e1bea.png)
It also doesn't support inline display, though it is functional.

**Comment by tronical** [comment]
That's true. The GL backend uses winit, which does not provide API to permit applications to respond to input method queries. But the IME position is something that we can and should set. Thanks for the report and the screenshot :)

**Comment by tronical** [comment]
This is also reproducible on macOS.

**Comment by tronical** [comment]
This is fixed via https://github.com/slint-ui/slint/pull/1713
